pkg/plugin/aggregation/run.go

```diff
@@ -189,6 +217,11 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 	}
 }
 
+func canStartPlugin(a *Aggregator, p plugin.Interface) bool {
```

Comment:
Even though the logic is small I wanted to put it here so that it would be easier to test/manipulate in the future.